src/DependencyInjection/KnpUOAuth2ClientExtension.php

```diff
@@ -248,6 +251,11 @@ private function configureProviderAndClient(ContainerBuilder $container, $provid
             $redirectRoute,
             $redirectParams,
         ]);
+        if (is_string($httpClient)) {
```

Comment:
Usually we do things like `if (null !== $httpClient)` or just `if ($httpClient)`. It's a bit weird to check for a string specifically.

Reply:
I'll do it, but it's recommended to do strict comparison. I took the habit with PHPStan. :wink: